doc/api.rst

```diff
@@ -180,6 +180,16 @@ Here is how you can attach a file to an experiment (or item):
     files = {'file': open('report.xls', 'rb')}
     print(manager.upload_to_experiment(94, files))
     print(manager.upload_to_item(17, files))
+    
+Fetch an uploaded file
+^^^^^^^^^^^^^^^^^^^^^^
+
+You can find the upload ID from the info of an experiment/item using the methods described above.
+
+.. code-block:: python
+    
+    # This example gets a file with upload ID 63
+    print(manager.get_upload(63))
```

Comment:
I think print here is a bad example, as the answer is a file, it would be better to provide an example that saves the file:

Untested snippet:

~~~python
with open('my_file', 'w') as file:
    file.write(manager.get_upload(63))
~~~

Reply:
I was checking this out. The issue here is that elabapy calls the requests.json() function when it gets the data. I was working with .json files and everything works smooth with those. I get a python dictionary that I can print or write to file. But for other files I tried, the JSON decoder obviously fails as it can't handle their formatting.

I think would it would be better to not make elabapy treat the response as a json object. 

Once that is clear, the example can better reflect functionality.

Reply:
yeah, we probably need to have in baseapi.py another function to get binary data, and then the get_upload will call that. In fact `get_data` should be renamed to `get_json`.